client/src/models/Organization.js

```diff
@@ -159,14 +159,18 @@ export default class Organization extends Model {
     return this.shortName || this.longName || this.identificationCode
   }
 
-  static toIdentificationCodeString(organization) {
-    return organization.type === Organization.TYPE.PRINCIPAL_ORG
-      ? `${organization.shortName} \\ ${
-        Settings.fields.principal.org.identificationCode?.label
-      }: ${organization.identificationCode || "Not specified"}`
-      : `${organization.shortName} ${organization.longName} ${
-        organization.identificationCode || ""
-      }`
+  toIdentificationCodeString() {
```

Comment:
This method is never used, only the static version is called; so this can be removed.